### PR TITLE
5176: Edit org confirmation page

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/helpers/contextCreatorHelper.test.js [hunk 1]
```diff
@@ -35,4 +35,146 @@ describe('context creator helpers', () => {
       expect(response.includes(null)).toBeFalsy();
     });
   });
+
+  describe('addParamsToManifest', () => {
```
Comment (kat1906) [inline app/helpers/contextCreatorHelper.test.js:39]: moved into this file - no changes have been made

File: app/helpers/contextCreatorHelper.js [hunk 2]
```diff
@@ -2,3 +2,9 @@ export const extractObjectValuesToArray = obj => Object.values(obj).reduce((acc,
   if (value) acc.push(value);
   return acc;
 }, []);
+
+export const addParamsToManifest = (json, params) => JSON.parse(
```
Comment (alibombaye) [inline app/helpers/contextCreatorHelper.js:6]: good move 👍

**Comment by kat1906** [comment]
I forgot ui tests - will add now (everything else should be done)

File: app/routes.js [hunk 3]
```diff
@@ -99,7 +106,7 @@ export const routes = (authProvider) => {
       accessToken,
       data: req.body,
     });
-    return res.render('pages/adduser/template', addContext({ context, user: req.user }));
+    return res.render('pages/adduser/template', addContext({ context, user: req.user, csrfToken: req.csrfToken() }));
```
Comment (alibombaye) [inline app/routes.js:109]: 😂

File: app/pages/editorg/ui.test.js [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+  await pageSetup(t, true);
+  await t.navigateTo(pageUrl);
+
+  const editOrgPage = Selector('[data-test-id="edit-organisation-page"]');
+
+  await t
+    .expect(editOrgPage.exists).ok();
+});
+
+test('should navigate to /organisations/org when click on Back', async (t) => {
```
Comment (alibombaye) [inline app/pages/editorg/ui.test.js:69]: navigate to `/organisations/org1`

File: app/pages/editorg/ui.test.js [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+  await pageSetup(t, true);
+  await t.navigateTo(`${pageUrl}/confirmation`);
+
+  const editOrgComfirmationPage = Selector('[data-test-id="edit-organisation-confirmation-page"]');
+
+  await t
+    .expect(editOrgComfirmationPage.exists).ok();
+});
+
+test('should navigate to /organisations/org when click on Back', async (t) => {
```
Comment (alibombaye) [inline app/pages/editorg/ui.test.js:239]: navigate to `/organisations/org1`

**Pull request merged**